grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -541,12 +541,19 @@ class ApiController extends BaseController {
             ],
             parameters = [
                     @Parameter(name = "opusId",
+                            in = ParameterIn.PATH,
                             required = true,
                             description = "Collection id - UUID or short name"),
                     @Parameter(name = "profileId",
+                            in = ParameterIn.PATH,
                             required = true,
                             description = "Profile id - UUID or Scientific name"),
+                    @Parameter(name = "imageId",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "Image id - UUID or Scientific name"),
```

Comment:
imageId is only UUID. scientific name is not added.

Reply:
changed